services/common/src/tests/handlers.ts

```diff
@@ -43,6 +43,52 @@ const permitHandlers = [
 
 ]
 
+
+const permitSearchHandlers = [
+  http.post(
+    `/%3CAPI_URL%3E/search/permit-conditions`,
+    async ({ request, params }) => {
+      const requestBody = await request.json() as { query: string };
+
+      // Mock different responses based on search query
+      if (requestBody?.query?.includes('water')) {
+        return HttpResponse.json(
+          {
+            documents: [
```

Comment:
might prefer for this to be in data mocks, and maybe given a type (or types) so that any changes to the interface prompt an error